### PR TITLE
Add (optional) dependencies of newer tpot

### PR DESCRIPTION
**Pull request by bollwyvl**
Been meaning to get around to this for some time, but redid them from scratch...

### CODE REVIEW

**Comment by bollwyvl** [comment]
Great, `tpot` and `update_checker` are already there. Will focus on these, and then PR conda-forge/tpot-feedstock with newer stuff

File: recipes/skrebate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/skrebate/meta.yaml]: Also possibly `noarch: python`?

File: recipes/skrebate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+test:
+  imports:
+    - skrebate
+
+about:
+  home: https://epistasislab.github.io/scikit-rebate
+  license: MIT
+  license_family: MIT
+  # TODO: upstream a PR to add LICENSE to MANIFEST.in
+  # license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/skrebate/meta.yaml]: Please submit the PR and list it as a comment in this recipe. Also:

conda-forge.github.io/docs/meta.html#packaging-the-license-manually

File: recipes/stopit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+test:
+  imports:
+    - stopit
+
+about:
+  home: https://pypi.python.org/pypi/stopit
+  license: GPL-3.0
+  license_family: GPL
```
Comment (synapticarbors) [inline recipes/stopit/meta.yaml]: Could you submit an issue upstream asking them to include a license file in the source distribution and put a link to the issue as a comment here.

File: recipes/stopit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/stopit/meta.yaml]: Possibly `noarch: python`

File: recipes/stopit/meta.yaml (hunk 3 above)
Comment (bollwyvl) [reply recipes/stopit/meta.yaml]: [PR](https://github.com/glenfant/stopit/pull/15)!

File: recipes/skrebate/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipes/skrebate/meta.yaml]: [PR](https://github.com/EpistasisLab/scikit-rebate/pull/31)!

File: recipes/scikit-mdr/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+  description: |
+    MDR is an effective feature construction algorithm that is capable of
+    modeling higher-order interactions and capturing complex patterns in data
+    sets.
+
+    MDR currently only works with categorical features and supports both binary
+    classification and regression problems. We are working on expanding the
+    algorithm to cover more problem types and provide more convenience features.
+  doc_url: http://simplejson.readthedocs.io/
```
[the remark this answers is not in the document]
Comment (bollwyvl) [reply recipes/scikit-mdr/meta.yaml]: got it

File: recipes/scikit-mdr/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
[the remark this answers is not in the document]
Comment (bollwyvl) [reply recipes/scikit-mdr/meta.yaml]: works locally!

File: recipes/skrebate/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/skrebate/meta.yaml]: also works locally

File: recipes/stopit/meta.yaml (hunk 4 above)
Comment (bollwyvl) [reply recipes/stopit/meta.yaml]: works locally

File: recipes/scikit-mdr/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    MDR currently only works with categorical features and supports both binary
+    classification and regression problems. We are working on expanding the
+    algorithm to cover more problem types and provide more convenience features.
+  dev_url: https://github.com/EpistasisLab/scikit-mdr
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - bryketos
+    - EpistasisLab
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: at request of @rhiever, i've added @EpistasisLab as a maintainer, which is an organization (in this case, a lab at Penn)

https://github.com/EpistasisLab/scikit-rebate/pull/31#issuecomment-354143469

This does seem like a great feature for certain cases, but maybe not in all, depending on how the org in question is structured... figured I'd let CI chew on it, but any formal guidance would be good!

Comment (rhiever) [reply recipes/scikit-mdr/meta.yaml]: I can provide a list of specific GitHub names to use in place of the org if needed/desired.

**Comment by jakirkham** [comment]
We don’t have support for listing orgs as maintainers currently.

File: recipes/scikit-mdr/meta.yaml (hunk 7 above)
Comment (rhiever) [reply recipes/scikit-mdr/meta.yaml]: List:

* @lacava
* @pschmitt52
* @rhiever
* @ryanurbs
* @weixuanfu

Comment (bollwyvl) [reply recipes/scikit-mdr/meta.yaml]: updated with list given.

**Comment by bollwyvl** [comment]
yeah, didn't even know how that might work, as i don't think an org can join a team, for example. either way, that's a good number of people!

**Comment by synapticarbors** [comment]
@bollwyvl - can you confirm that all of the people listed as co-maintainers are ok being listed? If they are, I'm happy to merge.

File: recipes/scikit-mdr/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+    MDR currently only works with categorical features and supports both binary
+    classification and regression problems. We are working on expanding the
+    algorithm to cover more problem types and provide more convenience features.
+  dev_url: https://github.com/EpistasisLab/scikit-mdr
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - lacava
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: @lacava, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?

File: recipes/scikit-mdr/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    - pschmitt52
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: @pschmitt52, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?

File: recipes/scikit-mdr/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+    - rhiever
+    - ryanurbs
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: @ryanurbs, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?

File: recipes/scikit-mdr/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+    - weixuanfu
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: @weixuanfu, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?

File: recipes/scikit-mdr/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+    classification and regression problems. We are working on expanding the
+    algorithm to cover more problem types and provide more convenience features.
+  dev_url: https://github.com/EpistasisLab/scikit-mdr
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - lacava
+    - pschmitt52
+    - rhiever
```
Comment (bollwyvl) [inline recipes/scikit-mdr/meta.yaml]: @rhiever, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?

Comment (rhiever) [reply recipes/scikit-mdr/meta.yaml]: Fine with me!

File: recipes/scikit-mdr/meta.yaml (hunk 11 above)
Comment (weixuanfu) [reply recipes/scikit-mdr/meta.yaml]: Fine with me! Thank you!

**Comment by bollwyvl** [comment]
Two out of five ain't bad... and it's easy to add folks on small PRs after the fact. Will pull the non-repliers tomorrow some time... 

🔔 @ryanurbs @pschmitt52 @lacava

File: recipes/scikit-mdr/meta.yaml (hunk 8 above)
Comment (lacava) [reply recipes/scikit-mdr/meta.yaml]: sure!

**Comment by bollwyvl** [comment]
Ok, updated with those maintainers who confirmed for `skrebate` and `scikit-mdr`, and a `license_file` for `stopit`. Seems pretty okay!

**Comment by synapticarbors** [comment]
Great. I'll merge once the CI finishes.

**Comment by bollwyvl** [comment]
Huzzah!

**Pull request merged**